ffi/capi/src/pluralrules.rs

```diff
@@ -129,7 +132,6 @@ pub mod ffi {
                 &fixed_decimal::Decimal::try_from_utf8(s)?,
```

Comment:
while we're here, it's weird that there's a shortcut for the string -> FixedDecimal -> PluralOperands route, but not for the int -> FixedDecimal -> PluralOperands or float -> FixedDecimal -> PluralOperands route. I would be confused if the *only* constructor (when the decimal feature is turned off) would be from a string, for something that is inherently numeric.

Reply:
Done. I didn't add it for float since that needs FixedDecimal and ryu